docker/Dockerfile.mpich314_intel

```diff
@@ -0,0 +1,166 @@
+##
```

Comment:
These Dockerfiles look good. Do you want to adapt all of them in order to make sure that they have a consistent form?

Reply:
I can but I seriously doubt that this will be the final form of them after I incorporate the changes required for the fv3gfs-python builds.  The goal now is just to add something that builds and can be used for simple testing

Reply:
Ok. You call. I guess it depends how much work it would be. If you think it's feasible without too much effort, I would go for refactoring all of them.